src/middleware/aws/operations/dynamodb.rs

```diff
@@ -14,12 +14,14 @@ impl<'a> AwsSpanBuilder<'a> {
             table_names.into_iter().map(|item| item.into()).collect();
         let mut attributes = vec![
             KeyValue::new(semconv::DB_SYSTEM, "dynamodb"),
+            #[allow(deprecated)]
             KeyValue::new(semconv::DB_OPERATION, method.clone()),
         ];
         match table_names.len() {
             0 => {}
             1 => {
                 attributes.extend([
+                    #[allow(deprecated)]
```

Comment:
We should try to replace deprecated attributes with up to date ones, but Splunk may not have support for them yet

Reply:
I'm 90% sure Splunk doesn't support new attributes, so let's use both for now and then attempt to remove them later